.github/workflows/firebase-hosting-pull-request.yml

```
@@ -7,17 +7,27 @@ jobs:
         runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: ./packages/chat
         steps:
-            - uses: actions/checkout@v2
-            - name: Setup Hugo
-              uses: peaceiris/actions-hugo@v2
+            - uses: actions/checkout@v4
+            - name: Setup Node.js
+              uses: actions/setup-node@v3
               with:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 17: did not find expected key

(syntax-check)